docs/src/tutorials/object_detection.md

```diff
@@ -0,0 +1,122 @@
+# Object Detection using HOG
+
+In this tutorial, we will use Histogram of Oriented Gradient (HOG) feature descriptor based linear SVM to create a person detector. We will first create
+a person classifier and then use this classifier with a sliding window to identify and localize people in an image.
+
+The key challenge in creating a classifier is that it needs to work with variations in illumination, pose and occlusions in the image. To achieve this, we will train
+the classifier on an intermediate representation of the image instead of the pixel-based representation. Our ideal representation (commonly called feature vector)
+captures information which is useful for classification but is invariant to to small changes in illumination and occlusions. HOG descriptor is a gradient-based
+representation which is invariant to local geometric and photometric changes (i.e. shape and illumination changes) and so is a good choice for our problem. In fact
+HOG descriptors are widely used for object detection.
+
+We will use [this data](https://drive.google.com/file/d/0B9V0KF3ZHWtWa0hGNHJDSm1XZnM/view?usp=sharing) for training our classifier. We will start by loading the data and computing HOG features of all the images.
+
+```julia
+using Images, ImageFeatures
+
+pos_examples = "path_to_data/human/"
+neg_examples = "path_to_data/not_humans/"
+
+n_pos = length(readdir(pos_examples))   # number of positive training examples
+n_neg = length(readdir(neg_examples))   # number of negative training examples
+n = n_pos + n_neg                       # number of training examples 
+data = Array{Float64}(n, 3780)          # Array to store HOG descriptor of each image. Each image in our training data has size 128x64 and so has a 3780 length 
```

Comment:
I suspect this should be `Array{Float64}(3780, n)` (the transpose of what you have).